### ISSUE TITLE
nu@0.40.0: Decompress Error

### ISSUE THREAD

**Issue by igitur** [ScoopInstaller/Main#3003]
Could not find 'nushell-0.40.0 in D:\scoop\apps\nu\0.40.0\nu_0_40_0_windows'! 

```
>scoop install nu
Installing 'nu' (0.40.0) [64bit] [main]
WARN  By installing you accept following license: MIT (https://spdx.org/licenses/MIT.html)
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 0874f6|OK  |   4.2MiB/s|D:/scoop/cache/nu#0.40.0#https_github.com_nushell_nushell_releases_download_0.40.0_nu_0_40_0_windows.zip
Download: Status Legend:
Download: (OK):download completed.
Checking hash of nu_0_40_0_windows.zip ... ok.
Extracting nu_0_40_0_windows.zip ... ERROR Could not find 'nushell-0.40.0 in D:\scoop\apps\nu\0.40.0\nu_0_40_0_windows'! (error 11)

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=nu%400.40.0%3a+Decompress+Error&body=Could+not+find+%27nushell-0.40.0+in+D%3a%5cscoop%5capps%5cnu%5c0.40.0%5cnu_0_40_0_windows%27!+(error+11)
ERROR This application failed to install: nu
```

**Comment by rashil2000** [comment]
Unable to reproduce. Try uninstalling, clearing cache and installing again.

**Comment by goostleek** [comment]
It's definitely an `aria2` download issue. I could reproduce the same error if using `aria2`. It's not recommended to use aria download anymore as [it's buggy](https://github.com/ScoopInstaller/Scoop/issues/2647#issue-367450546). I can successfully install `nu` when `aria2-enabled` flag is not enabled
```
scoop config aria2-enabled false
```

![image](https://user-images.githubusercontent.com/7780330/143242277-aa59b593-6c58-430d-8a63-3038ef8b62d0.png)

**Comment by igitur** [comment]
It turns out this is not an aria2c issue, but nu is being blocked by my antivirus.

> D:\scoop\apps\7zip\19.00\7z.exe, which attempted to access D:\scoop\apps\nu\0.40.0\nu_0_40_0_windows\nushell-0.40.0\nu_plugin_inc.exe. The Trojan named Artemis!94B6BD2F9168 was detected and deleted.

Hopefully it's a false positive, but I'll close this issue.
